tests/unit/workflows/test_workflows_tasks_submission.py

```diff
@@ -35,9 +60,40 @@ class DummyEng(object):
     pass
 
 
+def mock_get_rt_instance():
+
+    class MockRtInstance(object):
+        def create_ticket(self, *args, **kwargs):
+            return 1
+
+    return MockRtInstance()
+
+
 def test_add_note_entry_does_not_add_value_that_is_already_present():
     obj = StubObj({'public_notes': [{'value': '*Temporary entry*'}]}, {'core': 'something'})
     eng = DummyEng()
 
     assert add_note_entry(obj, eng) is None
     assert {'public_notes': [{'value': '*Temporary entry*'}]} == obj.data
+
+
+@mock.patch('inspirehep.modules.workflows.tasks.submission.get_instance',
+            side_effect=mock_get_rt_instance)
+def test_submit_rt_ticket(mocker_rt_instance):
+    """test that a ticket is sent to the RT service"""
+
+    body = ''
+    queue = ''
+    context = {}
+    obj = MockObj(1, {}, {})
+    ticket_id_key = 'ticket_id'
+    current_app.config['PRODUCTION_MODE'] = True
```

Comment:
And this is another way to do it, but I like it less, despite the fact that I just wrote it ._. https://github.com/inspirehep/inspire-next/blob/cef4089e2c381f00bcd0631e9f388db30b6bb680/tests/unit/workflows/test_workflows_tasks_actions.py#L164-L176